Mathlib/Data/List/Basic.lean

```diff
@@ -3752,14 +3752,8 @@ end Diff
 
 /-! ### enum -/
 
-theorem length_enumFrom : ∀ (n) (l : List α), length (enumFrom n l) = length l
```

Comment:
What happened with the naming convention here? Has Std taken on a different scheme to mathlib?

Reply:
The "new" statement is `(enumFrom n l).length = l.length`. The naming has always been variable here.

Reply:
There are plenty of examples of both alternatives all over Mathlib and Std; I agree we should do something about it, but not in this PR. :-)